libs/wingsdk/test/core/__snapshots__/connections.test.ts.snap

```diff
@@ -56,10 +56,88 @@ return class Handler {
           "sourceCodeLanguage": "javascript",
           "timeout": 60000,
         },
-        "type": "wingsdk.cloud.Function",
+        "type": "@winglang/sdk.cloud.Function",
       },
     ],
     "sdkVersion": "0.0.0",
+    "types": {
+      "@winglang/sdk.cloud.Api": {
+        "className": "Api",
+        "sourcePath": "<ABSOLUTE PATH>/api.inflight.js",
```

Comment:
Will this need to be resolvable across machines? If so it'd be useful for this to be a module-relative path instead. I think it's fine to iterate on this but just wanted to check to see how critical the portability of simulator.json is.

I guess we could eventually just plop an entire JSII manifest in here haha.

Reply:
> Will this need to be resolvable across machines?

I think with the changes we have planned to allow the simulator running across multiple processes, there's still an assumption of a "master" process (the one which started the simulation) that is responsible for initializing all of the resources -- at the moment I think that's the only process that needs to resolve these paths. But it's a good question, I think we we'll revise this format many times in the future